index.html

```diff
@@ -11771,9 +11771,10 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
-				<p>Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
+				<p>Authors MUST NOT use <code>aria-hidden</code> to hide root elements of a document. For instance, the <code>html</code> or <code>body</code> elements in an HTML document. Otherwise, authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
 				<p class="note">Authors are advised to use extreme caution and consider a wide range of disabilities when hiding visibly rendered content from assistive technologies. For example, a sighted, dexterity-impaired individual may use voice-controlled assistive technologies to access a visual interface. If an author hides visible link text "Go to checkout" and exposes similar, yet non-identical link text "Check out now" to the accessibility API, the user may be unable to access the interface they perceive using voice control. Similar problems may also arise for screen reader users. For example, a sighted telephone support technician may attempt to have the blind screen reader user click the "Go to checkout" link, which they may be unable to find using a type-ahead item search ("Go to…").</p>
 				<p class="note">At the time of this writing, <code><sref>aria-hidden</sref>="false"</code> is known to work inconsistently in browsers. As future implementations improve, use caution and test thoroughly before relying on this approach.</p>
+				<p>To prevent authors erroneously hiding entire window-rendered documents only to those using assistive technology, user agents MUST ignore the attribute if it is specified on document root elements. For instance, in HTML the <code>html</code> and <code>body</code> elements. Or the root <code>svg</code> element if it is rendered as its own document. If authors were to specify <code>aria-hidden="true"</code> on the opening tag for an embedded document, for instance on a <code>math</code> or <code>svg</code> embedded within an HTML document, user agents would still be expected to hide these elements from assistive technologies.</p>
```

Comment:
Need to ask some HTML spec people if "rendered as its own document" is the right way to phrase that. I suspect there may be some other defined term.

In addition to `<math>` and `<svg>` should we consider whether to allow this on docs contained in an iframe? Is "embedded" the right term for those elements and containers? How does this apply to SVG in the context of an `<img src="foo.svg">`? Is the `src` reference "embedded" or "rendered as its own document"? Note: I think WebKit may still the only implementation that will try to pull some accessibility info (svg:title?) from the referenced SVG image, so there may be implementation-specific impact to whatever phrasing you land on.

Reply:
a comment i made on the ARIA wg call yesterday concerning iframes, is that i didn't think that should factor into whether the html/body elements should respect aria-hidden=true.  And that's because even if the browser detected that _this_ body element should be allowed to be hidden because it's within an iframe, that could still mean that the parent iframe was not hidden, and then there'd be an iframe available but no content within it.   Rather, it'd be better if there was consistency between browsers to completely hide iframes, and then if it / it's rendered content needed to be hidden, then the iframe would be the element to hide, not its nested document's html / body element.

Reply:
Same change as above.
```suggestion
				<p>To prevent authors erroneously hiding entire window-rendered documents only to those using assistive technology, user agents MUST not expose the hidden state to assistive technologies if it is specified on the root element or the host language element that <a href="https://html.spec.whatwg.org/multipage/dom.html#represents">represents</a> the contents of the primary document in view. For instance, the <code>html</code> or <code>body</code> elements in an HTML document. Or the root <code>svg</code> element if it is rendered as its own document. If authors were to specify <code>aria-hidden="true"</code> on the opening tag for an embedded document, for instance on a <code>math</code> or <code>svg</code> embedded within an HTML document, user agents would still be expected to hide these elements from assistive technologies.</p>
```